### PR TITLE
feat: add cloud native anti-ddos basic resource

### PR DESCRIPTION
**Pull request by ShiChangkuo**
**What this PR does / why we need it**:

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
add new resource **huaweicloud_antiddos_basic**

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST='./huaweicloud/services/acceptance/antiddos' TESTARGS='-run TestAccAntiDdos_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/antiddos -v -run TestAccAntiDdos_basic -timeout 360m -parallel 4
=== RUN   TestAccAntiDdos_basic
=== PAUSE TestAccAntiDdos_basic
=== CONT  TestAccAntiDdos_basic
--- PASS: TestAccAntiDdos_basic (112.72s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/antiddos  112.862s
```

### CODE REVIEW

File: docs/resources/antiddos_basic.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+subcategory: "Anti-DDoS"
+---
+
+# huaweicloud_antiddos_basic
```
Comment (niuzhenguo) [inline docs/resources/antiddos_basic.md:5]: basic is an adj, suggest to use a noun here.

Comment (ShiChangkuo) [reply docs/resources/antiddos_basic.md:5]: basic can also be a noun, we can get the product name from [service overview](https://support.huaweicloud.com/intl/en-us/productdesc-ads/ads_01_0001.html)
```
HUAWEI CLOUD provides multiple security solutions to defend against Distributed Denial-of-service (DDoS) attacks. 
You can select an appropriate one based on your service requirements. HUAWEI CLOUD Anti-DDoS Service (ADS) 
includes Cloud Native Anti-DDoS Basic (CNAD Basic), Cloud Native Anti-DDoS Advanced (CNAD Advanced), and
Advanced Anti-DDoS (AAD).
```

Comment (ShiChangkuo) [reply docs/resources/antiddos_basic.md:5]: basic can also be a noun, we can get the product name from [service overview](https://support.huaweicloud.com/intl/en-us/productdesc-ads/ads_01_0001.html)
```
HUAWEI CLOUD provides multiple security solutions to defend against Distributed Denial-of-service (DDoS) attacks. 
You can select an appropriate one based on your service requirements. HUAWEI CLOUD Anti-DDoS Service (ADS) 
includes Cloud Native Anti-DDoS Basic (CNAD Basic), Cloud Native Anti-DDoS Advanced (CNAD Advanced), and
Advanced Anti-DDoS (AAD).
```

Comment (niuzhenguo) [reply docs/resources/antiddos_basic.md:5]: fair enough

**Pull request merged**